.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+name: Build
+
+on:
+  push:
+    # dependabot will create both branches and pull request
+    # only run on pull requests to avoid running twice and creating a release draft
+    branches-ignore:
+      - 'dependabot/**'
+  pull_request:
+
+concurrency:
+  # Only run once for the latest commit per ref and cancel other (previous) runs.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          cache: 'npm'
```

We might want to pass [`check-latest`](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#check-latest-version) here, so that the latest version of Node.js in the version range is used always.
```suggestion
          check-latest: true
          cache: npm
```